proto/sge/bet/bet.proto

```diff
@@ -167,10 +167,10 @@ message BetFulfillment {
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"bet_amount\""
   ];
-  // payout amount fulfilled by the participation
-  string payout_amount = 4 [
+  // payout_profit is the fulfilled profit by the participation.
+  string payout_profit = 4 [
```

Comment:
can we just keep it as payout. Payout itself means amount returned or profit returned.

Reply:
profit is different than the payout, the profit is the subtraction of the payout and bet amount, in other words, the pure amount that the winner receives without the bet amount.